lib/scaffold/default-config.js

```diff
@@ -24,34 +24,47 @@ function getDefaultConfig(options) {
     mkdirp.sync(defaultPath);
   }
 
-  var defaultServices = ['bitcoind', 'db', 'address', 'web'];
+  var defaultServices = [
+    'address',
+    'block',
+    'db',
+    'fee',
+    'header',
+    'mempool',
+    'p2p',
+    'timestamp',
+    'transaction',
+    'web'
+  ];
+
   if (options.additionalServices) {
     defaultServices = defaultServices.concat(options.additionalServices);
   }
 
+  var defaultDataDir = path.resolve(defaultPath, './data');
+
+  if (!fs.existsSync(defaultDataDir)) {
+    mkdirp.sync(defaultDataDir);
+  }
+
   if (!fs.existsSync(defaultConfigFile)) {
     var defaultConfig = {
-      network: 'livenet',
+      network: 'testnet',
```

Comment:
You're right, that was a bad idea. We also need to figure out how to help people upgrade if they already have a config file. (Or at least show a warning if a config file from an older version exists.) I'll look at that now.